{
  'test_type': 'runtime',
  'output': """
    Run time error 4: "Array index out of bounds"
  """,
  'should_fail': True
}
